pkg/console/assets/bindata.go

```diff
@@ -970,11 +970,11 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//     data/
-//       foo.txt
-//       img/
-//         a.png
-//         b.png
+//	data/
```

Comment:
the script is switching spaces with tabs for some reason.. please put it back to spaces in order to satisfy the linter